sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -57,10 +74,28 @@ final class ActionService(
   override final val componentType = Actions.name
 }
 
-final class ActionsImpl(_system: ActorSystem, services: Map[String, ActionService], rootContext: Context)
+private[javasdk] object ActionsImpl {
+  private[action] val log = LoggerFactory.getLogger(classOf[Action])
```

Comment:
I think we can remove this logger here and initialize it in the `def log` above. 
It's kind of confusing. 

btw, there is a direct reference to it on line 308, we should use `service.log`

Reply:
We also use it for things not related to a specific service below, so it can't be in a specific service.

Reply:
Should perhaps be `classOf[ActionsImpl]` though

Reply:
ok and it makes sense to use ActionsImpl.